examples/developer-tools/src/blocks/inputs.ts

```diff
@@ -0,0 +1,149 @@
+import * as Blockly from 'blockly';
+
+const fieldMessage = 'fields %1 %2';
```

Comment:
This confused me for a sec. It looks like this is just being separated out to remove code duplication? Personally I think it's better to duplicate code in block definitions to aid readability, since they're declarative and change so infrequently. Just like unit tests.